src/script/cpp_api/s_env.h

```diff
@@ -50,7 +52,10 @@ class ScriptApiEnv : virtual public ScriptApiBase
 	void triggerLBM(int id, MapBlock *block,
 		const std::unordered_set<v3s16> &positions, float dtime_s);
 
+	static int override_abm(lua_State * L, ServerEnvironment *env);
 private:
+	static LuaABM *readABM(lua_State *L, int abm_index, int id);
```

Comment:
static functions should follow the snake case notation.
https://docs.luanti.org/for-engine-devs/code-style-guidelines/#function-declarations

```suggestion
	static LuaABM *read_abm(lua_State *L, int abm_index, int id);
```